smallrye-reactive-messaging-rabbitmq/src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1 @@
+junit.jupiter.execution.timeout.default=5 m
```

Comment:
It is per test or for the whole test suite? 

If it's per test that's awesome!

Reply:
The doc says "Default timeout for all testable and lifecycle methods". 
it is interesting because rabbitmq container startup happens on beforeall lifecycle method.

Now that we know it is the problem, we can apply a startup timeout with a retry. 
I'll need to check out if downgrading to the 3.11 image really resolves the issue. 
And then understand what is the issue with the newest rabbitmq images.